mw-webapp/src/App.tsx

```diff
@@ -1,6 +1,9 @@
 import {Header} from "src/component/header/Header";
 import {Router} from "src/router/Router";
 
+/**
+ * App
+ */
 function App() {
   return (
```

Comment:
Empty line before return

Reply:
{ "blankLine": "always", "prev": "*", "next": "return" }

I added this rule, but seems like ESLint do not count whole function as something and only add blankline between something inside it

Reply:
Added